.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "OpenTelemetry Java Instrumentation Dev Environment",
+  "build": {
+    "dockerfile": "Dockerfile",
+    "context": ".."
+  },
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "vscjava.vscode-java-pack",
+        "vscjava.vscode-gradle",
+        "ms-vscode.gradle"
+      ]
+    }
+  },
+  "features": {
+    "ghcr.io/devcontainers/features/java:1": {
+      "version": "21"
+    },
+    "ghcr.io/devcontainers/features/node:1": {
+      "version": "16"
+    }
+  },
+  "postCreateCommand": ".devcontainer/post-create.sh",
+  "remoteUser": "vscode",
+  "containerEnv": {
+    "GRADLE_USER_HOME": "/home/vscode/.gradle",
+    "GRADLE_OPTS": "-Xmx2g -XX:MaxMetaspaceSize=512m"
```

Comment:
let the JVM decide on Xmx

and MaxMetaspaceSize is already in the projects gradle.properties

```suggestion
```